src/validation_workflow/download_utils.py

```diff
@@ -0,0 +1,29 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+
+import requests
+
+from system.temporary_directory import TemporaryDirectory
+
+
+class DownloadUtils:
+    @staticmethod
+    def is_url_valid(url: str) -> int:
+        response = requests.head(url)
+        if response.status_code == 200:
+            return 1
+        else:
+            return 0
```

Comment:
Is there a benefit using 1/0 instead of True/False?
This is also opposite compares to normal programming pattern where 1 is false and 0 is true normally.

Plus, the `else` block can be completely removed in this case.